### ISSUE TITLE
Model checkpoints and achitechture

### ISSUE THREAD

**Issue by davidluciolu** [NVlabs/VideoITG#7]
Hello!

Thanks for your work!
I've got some questions:
- Which type of variant is your released checkpoint? It is Variant-C as depicted in your paper?
- Could you consider releasing the intermediate checkpoints, i.e. the pretrained model and the finetuned model (before training with VideoITG40k)?

Thanks again!

**Comment by exiawsh** [comment]
Hello! Sorry for the late response.

Yes — the released checkpoint is Variant-C as depicted in our paper, and it’s the highest-accuracy variant.
For the pretrained model, please refer to:
https://huggingface.co/exiawsh/eagle-qwen2-7b-finetune-uni-ov-video-finetune-sftv1
Thanks again for your interest and support!

@davidluciolu

**Comment by davidluciolu** [comment]
Thanks!
